apps/webcert/src/components/header/WebcertHeader.tsx

```diff
@@ -32,6 +39,8 @@ const WebcertHeader: React.FC<Props> = ({ isEmpty = false }) => {
   const dispatch = useDispatch()
   const loggedInCareProvider = useSelector(getLoggedInCareProvider)
   const careProviders = user && user?.careProviders
+  const unitName = user && user.loggedInUnit.unitName
+  const resourceLink = useSelector(getUserResourceLink(ResourceLinkType.WARNING_NORMAL_ORIGIN))
```

Comment:
instead of the `displayWarningNormalOriginBanner` function we can assign it here. `const displayWarningNormalOriginBanner = useSelector(getUserResourceLink(ResourceLinkType.WARNING_NORMAL_ORIGIN))`